test/test_device.py

```diff
@@ -87,7 +87,7 @@ def do_command(self, args):
         for arg in args:
             cli_args.append(shlex.quote(arg))
         if self.interface == 'cli':
-            proc = subprocess.Popen(['hwi ' + ' '.join(cli_args)], stdout=subprocess.PIPE, shell=True)
+            proc = subprocess.Popen(['../hwi.py ' + ' '.join(cli_args)], stdout=subprocess.PIPE, shell=True)
```

Comment:
`pip install -e .[tests]` wasn't working on my system and this was the only way I could run the unittests. Should I remove this?

Reply:
Maybe make that a seperate commit?

Reply:
Did you try using poetry to install the it and use its virtualenv?